core/domain/stats_domain.py

```diff
@@ -95,25 +117,45 @@ def get_multi(cls, exploration_id, rule_data):
 
     @classmethod
     def get(cls, exploration_id, state_name, rule_str):
-        # TODO(sll): Deprecate this method.
+        """Gets domain object corresponding to given rule spec strings and
```

Comment:
Gets the StateRuleAnswerLog domain object corresponding to the given ...

Also -- rule spec strings? Or just a single one? This docstring suggests plural; arg suggests singular.